exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,13 @@
 let name = "Daniel";
 let danielsRole = "mentor";
 
+if (danielsRole){
+console.log(`Hi, I'm ${name}, I'm a ${danielsRole}.`);
+}
+else if (Daniel){
+console.log(`Hi I'm ${name}, I'm a student`);
+}
+
```

Comment:
this is great Pamela. Mine is sooooo long and messy: Just look but dont laugh at me.....: 
let name = "daniel";
let danielsRole = "mentor";
function checkNameandRole(name, danielsRole) {
  if (name === "Daniel" && danielsRole === "mentor") {
    return "Hi, I'm Daniel, I'm a mentor";
  } else if (name === "Daniel" && danielsRole === "student") {
    return "Hi, I'm Daniel, I'm a student.";
  } else {
    return "I am superwoman";
  }
}